kimchi/src/circuits/expr.rs

```diff
@@ -511,6 +511,10 @@ pub enum FeatureFlag {
     LookupTables,
     RuntimeLookupTables,
     LookupPattern(LookupPattern),
+    /// Enabled if the table width is at least the given number
+    TableWidth(isize), // NB: isize so that we don't need to convert for OCaml :(
+    /// Enabled if the number of lookups per row is at least the given number
+    LookupsPerRow(isize), // NB: isize so that we don't need to convert for OCaml :(
```

Comment:
ocaml-gen now supports `usize <-> int` so feel free to use `usize`

Reply:
https://github.com/o1-labs/ocaml-gen/commit/586deaa0bd615e3326d4b1bb7396d57ceffbbbce

Reply:
not sure I understand these feature flags at this point, is this going to panic if you try to create a pattern that does too many lookup per rows? or if you try to create a table that's too large?